src/lib/adapters/video/replicateVideoHandler.ts

```
@@ -0,0 +1,178 @@
+    const inputPayload: Record<string, unknown> = {
+      image: dataUri,
+      prompt,
+      num_frames: (options.length ?? 4) * 24, // Assume 24 fps
+      fps: 24,
+      aspect_ratio: options.aspectRatio,
+      ...(options.resolution !== undefined
+        ? { resolution: options.resolution }
+        : {}),
+    };
+
+    let prediction;
```

Biome (2.4.15)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)